docs/config.md

```diff
@@ -83,6 +83,7 @@ This is a list of available options:
 | `core.autoimport`      | `bool`   | Import missing keys stored in the pass repository without asking. | `false` |
 | `core.autosync`        | `bool`   | Always do a `git push` after a commit to the store. Makes sure your local changes are always available on your git remote. | `true` |
 | `core.cliptimeout`     | `int`    | How many seconds the secret is stored when using `-c`. | `45` |
+|  `core.clear-after` | `bool` | This setting determines wheter or not the clipboard is cleared after `gopass -c`. Setting this to `false` takes precedence over the value of `core.cliptimeout` | `true` |
```

Comment:
Isn't setting `core.cliptimeout` to 0 doing that already?

Reply:
No. We're using a default of 45 in that case.

https://github.com/gopasspw/gopass/blob/master/pkg/clipboard/clipboard.go#L47